packages/generators/templates/tsconfig.json.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	compilerOptions: {
+		noImplicitAny: true,
+		module: "es6",
+		target: "es5",
+		jsx: "react",
```

Comment:
I'd remove it. We don't know what the user will use for their project. 
We could set `allowSyntheticDefaultImports` to `true`